Resources/ServerInfo/Guidebook/Engineering/PneumaticValve.xml

```diff
@@ -0,0 +1,40 @@
+﻿<Document>
+  # Pneumatic Valve
+  The pneumatic valve is a bidirectional valve controlled via a pressure input.
+  <Box>
+    <GuideEntityEmbed Entity="PressureControlledValve"/>
+  </Box>
+
+  The pneumatic valve has 3 connections: input, output, and control. The "input" side will be the input/output connection with the highest pressure, and can switch sides, making the valve bidirectional.
+
+  <Box>
+    <GuideEntityEmbed Entity="GasPipeStraight" Caption="Input/Output"/>
+  </Box>
+  <Box>
+    <GuideEntityEmbed Entity="GasPipeStraight" Caption="Control" Rotation="90"/>
+    <GuideEntityEmbed Entity="PressureControlledValve"/>
+    <GuideEntityEmbed Entity="FloorTileItemSteel" Caption=""/>
+  </Box>
+  <Box>
+    <GuideEntityEmbed Entity="GasPipeStraight" Caption="Input/Output"/>
+  </Box>
+
+  The valve will [color=green]open[/color] when the pressure on the output side is lower than the pressure on the control side by 101.35 kPa.
+
+  The valve will [color=red]close[/color] when the pressure of the output side reaches the pressure of the control side within 101.35 kPa.
+
+  For example, a pneumatic valve with a control pressure of 500 kPa will open when the output pressure is 398.65 kPa or lower, and it will close when the output pressure is 398.65 kPa or higher.
+
+  The valve's control pressure is determined by a pipenet connection, and as such can be adjusted on the fly by a pump or another source of pressure control.
+
+  ## Differences to Pumps
+
+  The pneumatic valve is different from a pump which moves gas via work. The pneumatic valve is a passive device that moves gas based on the higher pressure of the input gas, and as such it can sometimes fill volumes faster than a pump can.
+
+  For example, a pneumatic valve with a control pressure of 500 kPa will fill a volume faster than a pressure pump set to 500 kPa. However, the pump will be able to maintain the pressure in the volume more accurately.
```

Comment:
nope pressure pumps are instant

Reply:
so I guess I worded it wrong(?) or im just completely wrong, but the pneumatic valve moves gas based on this:

                transferRate = Math.Min(control * comp.Gain, comp.MaxTransferRate * _atmosphereSystem.PumpSpeedup());

where the "comp.Gain" is just a constant of 10, so this would work out to "your transfer rate is the difference between your target and your output multiplied by 10, OR 200L, whichever is less."

Because of this I frequently use pneumatic valves for volume fills, because they'll work faster compared to a pump set to 500kPa (because it will move gas in slower 500kPa "increments").

Am I right, or have I just misinformed myself?

Reply:
it depends on situation, pneumatic valves are kinda cursed and transfer volume dependent on the dP of the lower-pressure side with the control side

Reply:
How does PumpSpeedup work again? Last I checked in the code I thought pump speed up was an "overclock" mode that caused gas to leak from the device but it would speed up operation.